.envrc

```
@@ -6,10 +6,17 @@
 
 # React configs
 
-export REACT_APP_OKTA_ISSUER=https://test.idp.idm.cms.gov
+export REACT_APP_OKTA_DOMAIN=https://test.idp.idm.cms.gov
+export REACT_APP_OKTA_SERVER_ID=aus2e96etlbFPnBHt297
+export REACT_APP_OKTA_ISSUER=$REACT_APP_OKTA_DOMAIN/oauth2/$REACT_APP_OKTA_SERVER_ID
 export REACT_APP_OKTA_REDIRECT_URI=http://localhost:3000/implicit/callback
 export REACT_APP_OKTA_CLIENT_ID=0oa2e913coDQeG19S297
 
+
+# OKTA variables needed for  
+export OKTA_ISSUER=$REACT_APP_OKTA_ISSUER
+export OKTA_CLIENT_ID=0oa2e913coDQeG19S297
```

`REACT_APP_OKTA_CLIENT` and `OKTA_CLINET_ID` are the same. Can we reference them the same way we do for `OKTA_ISSUER`

```suggestion
export OKTA_CLIENT_ID=$REACT_APP_OKTA_CLIENT_ID
```

---

I think this should actually be backwards, as the regular env variable is the source of truth for the rest.
`export REACT_APP_OKTA_CLIENT_ID=$OKTA_CLIENT_ID`